tests/tests/test_import.py

```diff
@@ -297,3 +300,45 @@ def test_import_page_with_rich_text_link(self):
         self.assertEqual(page.body, '<p>But I have a <a id="1" linktype="page">link</a></p>')
 
         # TODO: this should include an embed type as well once document/image import is added
+
+    def test_import_collection(self):
+        root_collection = Collection.objects.get()
+
+        IDMapping.objects.get_or_create(
+            uid="f91cb31c-1751-11ea-8000-0800278dc04d",
+            defaults={
+                'content_type': ContentType.objects.get_for_model(Collection),
+                'local_id':  root_collection.id,
+            }
+        )
```

Comment:
We can't safely assume that an IDMapping exists for the root collection on the destination site; as it stands, if a mapping doesn't exist then it will fail at the point it attempts to create the root node by calling `parent.add_child` on a non-existent parent. I think the cleanest way to resolve that is to special-case nodes with depth=2 to not set the parent as a dependency, and on running the 'create' operation, fetch the parent with `get_first_root_node` instead of looking up by uid.

Reply:
Forgot about this, sorry. It was hacked for the demo